Blast, Mantle, and ZKsync - open beta message. (#1379)
MetaMask/metamask-docs: services/reference/blast/index.md

Blast is supported through the [DIN](https://www.infura.io/solutions/decentralized-infrastructure-service) service,
meaning calls to the network are routed to [partner infrastructure providers](#partners-and-privacy-policies).

> Infura provides Open Beta access to the Blast nwtork. During this period, there might be feature limitations. Performance issues are not expected, but they are possible as we optimize and stabilize the service.

Reviewer: ```suggestion
Infura provides Open Beta access to Blast. During this period, there might be feature limitations. Performance issues are not expected, but they are possible as we optimize and stabilize the service.
```

I think referring to the network just by its name is better for consistency with the rest of the page.

Blast is supported through the [DIN](https://www.infura.io/solutions/decentralized-infrastructure-service) service,
meaning calls to the network are routed to [partner infrastructure providers](#partners-and-privacy-policies).

> Infura provides public beta access to Blast, and there may be updates made to performance and
> method availability based on user feedback.

Reviewer: ```suggestion
Infura provides public beta access to Blast, and updates might be made to performance and
method availability based on user feedback.
```

Either wording should be fine.
